spec/models/ability_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Ability do
+
```

Comment:
Sure there is a reason. This way the code is more legible. I don't use them everywhere, but sometimes a certain file looks better. So, as I said it several times, i will not change my empty-line style.

You must understand that slavishly adhere to such styles is counterproductive. That is because this particular guy uses certain font size, certain editor coloring, specific editor settings, probably different IDE and OS. For him, such style may work. For me - it doesn't.

Did you noticed how I never try to impose my coding styles on you? I simply do not assume automatically, as apparently you do, that my coding style is the best one and all other people who do differently are, by definition, inferior programmers.